rustsec/src/error.rs

```diff
@@ -131,14 +131,6 @@ impl From<fmt::Error> for Error {
     }
 }
 
-#[cfg(feature = "git")]
-#[cfg_attr(docsrs, doc(cfg(feature = "git")))]
-impl From<git2::Error> for Error {
-    fn from(other: git2::Error) -> Self {
-        format_err!(ErrorKind::Repo, &other)
-    }
-}
-
 impl From<io::Error> for Error {
```

Comment:
So unfortunately `gix` doesn't have a singular Error type, but rather...hundreds of unique errors. This is a bit tedious, but does mean that all the places that throw errors now have a human (me) supplied contextual string for them rather than that info getting lost in the previous way of just straight converting between error types.